.github/workflows/validate_examples.yml

```diff
@@ -0,0 +1,15 @@
+name: Validate examples
+
+on: [pull_request]
+
+jobs:
+  verify-json-validation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Validate JSON
+        uses: docker://orrosenblatt/validate-json-action:latest
+        env:
+          INPUT_SCHEMA: /petrinet/petrinet_schema.json
+          INPUT_JSONS: /petrinet/examples/sir.json
```

Comment:
```suggestion
        uses: snapcart/json-schema-validator@v1.0.0
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
          json_schema: ./petrinet/petrinet_schema.json
          json_path_pattern: ./petrinet/examples/.*.json$
```

I found this other json validation github action that lets us use regex to check all json files against a schema (https://github.com/snapcart/json-schema-validator)

Reply:
Something about the dependencies between the two jobs isn't right, we get this:
![image](https://user-images.githubusercontent.com/5939016/232118984-517b19fd-e261-4284-97dc-97e4780de277.png)

Reply:
@bgyori check out the latest commit. I found the most feature complete JSON validator is this: https://github.com/GrantBirki/json-yaml-validate

One thing it doesn't support currently is the validation of the new `format` stuff in json schema 07, I made a fork real quickly and added this feature. We can use this for now and I'll package this  up into a PR to the main repo and get that added. (https://github.com/mehalter/json-yaml-validate/commit/f2de1cfd026eaa50f8f3e6a3aa3baae2a1bbe92d)

It seems to do great though! :D